test/extended/authentication.sh

```diff
@@ -77,16 +77,79 @@ echo "[INFO] Running extended tests"
 
 sleep 10
 
-cp test/extended/authentication/sync-schema1-group1.yaml ${BASETMPDIR}
-os::util::sed "s/LDAP_SERVICE_IP/${LDAP_SERVICE_IP}/g" ${BASETMPDIR}/sync-schema1-group1.yaml
-openshift ex sync-groups --sync-config=${BASETMPDIR}/sync-schema1-group1.yaml --confirm
-
-oc delete groups --all
-
-cp test/extended/authentication/sync-schema2-01.yaml ${BASETMPDIR}
-os::util::sed "s/LDAP_SERVICE_IP/${LDAP_SERVICE_IP}/g" ${BASETMPDIR}/sync-schema2-01.yaml
-openshift ex sync-groups --sync-config=${BASETMPDIR}/sync-schema2-01.yaml --confirm
-
-
-# Run the tests
-#LDAP_IP=${LDAP_SERVICE_IP} TMPDIR=${BASETMPDIR} ginkgo -progress -stream -v -focus="authentication: OpenLDAP" ${OS_OUTPUT_BINPATH}/extended.test
+schema=('rfc2307' 'ad')
```

Comment:
This is dependent on the previous pull and the real diff will become obvious once it goes in and the diff is between this and that.